docs/site/translation.md

```diff
@@ -86,14 +86,21 @@ Coverage for other languages is at comprehensive.
 <!-- I don't think we want to encourage this: ... if there is a need to have coverage at lower level in some locale,
 please file a ticket. [CLDR-18283](https://unicode-org.atlassian.net/browse/CLDR-18283) -->
 
+### DateTime formats
+
 #### Gregorian Calendar (Year First) calendar
 
 This is a variant of the Gregorian calendar whose formats always use year-month-day ordering and a 24-hour time cycle. See [Year First Calendar] for more details.
 *Note: the code is `iso8601`, but disregard that; it will be changed after submission.*
 
 <!-- [CLDR-18447](https://unicode-org.atlassian.net/browse/CLDR-18447) -->
 
-### DateTime formats
+#### New flexible format and interval patterns 🆕
```

Comment:
The emoji at the end adds a `-` to the anchor.  May be better moving it to the body of the section? Otherwise, it could cause a change in anchors in the future.

Reply:
I was thinking about that. It looks better in the header, but makes the heading link unstable.